src/core/annotation.js

```diff
@@ -897,6 +870,45 @@ class TextAnnotation extends Annotation {
   }
 }
 
+class FreeTextAnnotation extends Annotation {
+  constructor(parameters) {
+    super(parameters);
+
+    this.data.annotationType = AnnotationType.FREETEXT;
+    let dict = parameters.dict;
+
+    this.data.textColor = Util.createRgbColor(dict.getArray('TextColor'));
+    this.data.textStyle = dict.get('DS');
+    this.data.fontSize = dict.get('FontSize');
+    this.data.opacity = dict.get('CA');
+    this.data.richText = dict.get('RC');
```

Comment:
I think it's better to not include rich text processing in this patch. There is a separate issue for that at #2966 and we have to start simple here because fetching the appearance stream is already a bit difficult. Keeping the patch small will make it easier to catch any mistakes and to review, and rich text support is a nice-to-have that can be done in a follow-up patch.